modules/cco-ccoctl-deleting-sts-resources.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/uninstalling-cluster-aws.adoc
+
+[id="cco-ccoctl-deleting-sts-resources_{context}"]
+= Deleting AWS resources with the Cloud Credential Operator utility
+
+To clean up resources after uninstalling an {product-title} cluster with the Cloud Credential Operator (CCO) in manual mode with STS, you can use the CCO utility (`ccoctl`) to remove the AWS resources that `ccoctl` created during installation.
+
+.Prerequisites
+
+. Extract and prepare the `ccoctl` binary.
+. Install an {product-title} cluster with the CCO in manual mode with STS.
```

Comment:
Should these be bullets instead of numbers?

Reply:
Hm, I suppose they do not need to be done int that order now that I think about it - good eye!